src/components/TournamentList/TournamentList.js

```diff
@@ -12,9 +14,14 @@ export default class TournamentList extends Component {
 
   render() {
     return (
-        <section>
+      <section>
```

Comment:
Well, it's a great topic for discussion, but don't forget that we are using react of 16 version and render method can return list of items without wrapper (this additional wrappers really bad as for me).
F.e. this case when u are using this Tournament List in parent component by wrapping it in <Col xs={12} sm={6} md={10} lg={10}>...
It would be great to delete such redundant wrappers. OR you may say that this is an unjust one :)

Reply:
I don't forget about react 16, i'll update react and fix it

Reply:
Cool :)